daemon/daemon_unix.go

```diff
@@ -1712,15 +1712,13 @@ func (daemon *Daemon) setupSeccompProfile() error {
 
 // RawSysInfo returns *sysinfo.SysInfo .
 func (daemon *Daemon) RawSysInfo(quiet bool) *sysinfo.SysInfo {
-	var opts []sysinfo.Opt
+	var siOpts []sysinfo.Opt
 	if daemon.getCgroupDriver() == cgroupSystemdDriver {
-		rootlesskitParentEUID := os.Getenv("ROOTLESSKIT_PARENT_EUID")
-		if rootlesskitParentEUID != "" {
-			groupPath := fmt.Sprintf("/user.slice/user-%s.slice", rootlesskitParentEUID)
-			opts = append(opts, sysinfo.WithCgroup2GroupPath(groupPath))
+		if euid := os.Getenv("ROOTLESSKIT_PARENT_EUID"); euid != "" {
```

Comment:
FWIW; I initially wanted to add validation here, didn't see a clean way to return a warning from this function. Perhaps we need to make these functions return a `[]warnings` or `[]errors`, but I decided not to spend too much time on that; the `oci_linux` would now provide an error, so things won't go unnoticed at least.

Might still be good to look at improving these functions (in general: not this one specifically) to more easily allow warnings to be communicated to the user.